airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -649,21 +409,26 @@ public void testScheduledSync() throws Exception {
     final DestinationSyncMode destinationSyncMode = DestinationSyncMode.OVERWRITE;
     catalog.getStreams().forEach(s -> s.getConfig().syncMode(syncMode).destinationSyncMode(destinationSyncMode));
 
-    createConnection(connectionName, sourceId, destinationId, List.of(operationId), catalog, connectionSchedule);
+    var conn = createConnection(connectionName, sourceId, destinationId, List.of(operationId), catalog, connectionSchedule);
 
     // When a new connection is created, Airbyte might sync it immediately (before the sync interval).
     // Then it will wait the sync interval.
-    // todo: wait for two attempts in the UI
     // if the wait isn't long enough, failures say "Connection refused" because the assert kills the
     // syncs in progress
-    sleep(Duration.ofMinutes(4).toMillis());
+    List<io.airbyte.api.client.model.generated.JobWithAttemptsRead> jobs = new ArrayList<>();
+    while (jobs.size() < 2) {
+      var a = new io.airbyte.api.client.model.generated.JobListRequestBody().configTypes(List.of(JobConfigType.SYNC))
+          .configId(conn.getConnectionId().toString());
+      var resp = apiClient.getJobsApi().listJobsFor(a);
```

Comment:
```suggestion
      var listSyncJobsRequest = new io.airbyte.api.client.model.generated.JobListRequestBody().configTypes(List.of(JobConfigType.SYNC))
          .configId(conn.getConnectionId().toString());
      var resp = apiClient.getJobsApi().listJobsFor(listSyncJobsRequest);
```
nit to rename this variable

Reply:
oops! thanks!